From cafc72d3713e03b28ae14a7fe70de69e20ae44d7 Mon Sep 17 00:00:00 2001
From: Michal Rostecki <mrostecki@opensuse.org>
Date: Mon, 18 Mar 2019 15:20:43 +0100
Subject: [PATCH] bazel: transport sockets: Update grpc to 1.19.1

Risk Level: Low
Testing: Unit tests
Docs Changes: N/A
Release Notes: N/A

Signed-off-by: Michal Rostecki <mrostecki@opensuse.org>
---
 bazel/repository_locations.bzl                     | 6 +++---
 source/extensions/transport_sockets/alts/config.cc | 2 +-
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/bazel/repository_locations.bzl b/bazel/repository_locations.bzl
index e6863079f..824cfdd73 100644
--- a/bazel/repository_locations.bzl
+++ b/bazel/repository_locations.bzl
@@ -64,9 +64,9 @@ REPOSITORY_LOCATIONS = dict(
         urls = ["https://github.com/google/libprotobuf-mutator/archive/c3d2faf04a1070b0b852b0efdef81e1a81ba925e.tar.gz"],
     ),
     com_github_grpc_grpc = dict(
-        sha256 = "d99db0b39b490d2469a8ef74197d5f211fa740fc9581dccecbb76c56d080fce1",
-        strip_prefix = "grpc-1.16.0",
-        urls = ["https://github.com/grpc/grpc/archive/v1.16.0.tar.gz"],
+        sha256 = "f869c648090e8bddaa1260a271b1089caccbe735bf47ac9cd7d44d35a02fb129",
+        strip_prefix = "grpc-1.19.1",
+        urls = ["https://github.com/grpc/grpc/archive/v1.19.1.tar.gz"],
     ),
     com_github_nanopb_nanopb = dict(
         # From: https://github.com/grpc/grpc/blob/v1.14.0/bazel/grpc_deps.bzl#L123
diff --git a/source/extensions/transport_sockets/alts/config.cc b/source/extensions/transport_sockets/alts/config.cc
index c128562ee..708570d6c 100644
--- a/source/extensions/transport_sockets/alts/config.cc
+++ b/source/extensions/transport_sockets/alts/config.cc
@@ -85,7 +85,7 @@ createTransportSocketFactoryHelper(const Protobuf::Message& message, bool is_ups
     // Specifying target name as empty since TSI won't take care of validating peer identity
     // in this use case. The validation will be performed by TsiSocket with the validator.
     tsi_result status = alts_tsi_handshaker_create(
-        options.get(), target_name, handshaker_service.c_str(), is_upstream, &handshaker);
+        options.get(), target_name, handshaker_service.c_str(), is_upstream, nullptr, &handshaker);
     CHandshakerPtr handshaker_ptr{handshaker};
 
     if (status != TSI_OK) {
-- 
2.16.4

